extensions/standard-processors/processors/GenerateFlowFile.h

```diff
@@ -70,14 +66,15 @@ class GenerateFlowFile : public core::Processor {
       .withDefaultValue("Binary")
       .build();
   EXTENSIONAPI static constexpr auto UniqueFlowFiles = core::PropertyDefinitionBuilder<>::createProperty("Unique FlowFiles")
-      .withDescription("If true, each FlowFile that is generated will be unique. If false, a random value will be generated and all FlowFiles")
+      .withDescription("If true, each FlowFile that is generated will be unique. "
+          "If false, a random value will be generated and all FlowFiles will get the same content but this offers much higher throughput")
```

Comment:
all FlowFiles in a batch?

Reply:
All flowfiles period.
The description is directly from NiFi. And afaik nifi and minifi works in the same way so if this is false we only generate the date once. (except if CustomText is set then we should evaluate it once per batch)

Reply:
yes that is what I meant, so the description is not true

Reply:
as discussed the description of CustomText is clear enough on how to use that and what to expect